src/machine/machine_avr.go

```diff
@@ -58,3 +58,93 @@ func (p GPIO) Get() bool {
 		return (val > 0)
 	}
 }
+
+const (
+	COM2A1 = 7
+	COM2A0 = 6
+	COM2B1 = 5
+	COM2B0 = 4
+	COM0A1 = 7
+	COM0A0 = 6
+	COM0B1 = 5
+	COM0B0 = 4
+	COM1A1 = 7
+	COM1A0 = 6
+	COM1B1 = 5
+	COM1B0 = 4
+
+	CS00 = 0
+	CS01 = 1
+	CS11 = 1
+	CS22 = 2
+
+	WGM10 = 0
+	WGM11 = 1
+	WGM20 = 0
+	WGM21 = 1
+)
+
+// InitPWM initializes the registers needed for PWM.
+func InitPWM() {
+	// use waveform generation
+	*avr.TCCR0A |= avr.TCCR0A_WGM0
+
+	// set timer 0 prescale factor to 64
+	*avr.TCCR0B |= 1 << CS01
+	*avr.TCCR0B |= 1 << CS00
```

Comment:
Merging these two lines probably reduces code size, as writing to `*avr.TCCR0B` cannot be optimized (the pointer is marked volatile). This should be more compact and equivalent (but see the comment above regarding bit numbers):

```go
	*avr.TCCR0B |= (1 << CS01) | (1 << CS00)
```

Reply:
Merging into single line seems like a good idea.